app/templates/apps/list.html

```diff
@@ -2,6 +2,14 @@
 
 {% set page_title = "Apps" %}
 
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+  <ol>
+    <li><a href="/">Home</a></li>
+  </ol>
+</div>
+{% endblock %}
```

Comment:
Since this is the same in all three templates, you could put it in an include, or a macro

Reply:
Macro, of course. Thx. 👍